pkg/operator/workload/sync_openshift_oauth_apiserver.go

```diff
@@ -102,6 +107,12 @@ func (c *OAuthAPIServerWorkload) PreconditionFulfilled() (bool, error) {
 		klog.Info("Waiting for observed configuration to be available")
 		return false, errors.New("waiting for observed configuration to be available (haven't found APIServerArguments in spec.ObservedConfig.Raw)")
 	}
+
+	// specifying etcd servers list is mandatory, without it the pods will be crashlooping, so wait for it.
+	if storageServers, _ := operatorCfg.APIServerArguments[libgoetcd.StorageConfigURLsKey]; len(storageServers) == 0 {
```

Comment:
`storageServers := operatorCfg.APIServerArguments[libgoetcd.StorageConfigURLsKey; len(storageServers) == 0`
should work just the same, shouldn't it? (leaving the ignored variable out)

Reply:
yes, If the key doesn't exist we will get an empty slice :)